<launch>

  <arg name="nao_ip" default="$(optenv NAO_IP 192.168.1.127)"/>
  <arg name="nao_port" default="$(optenv NAO_PORT 9559)"/>

  <!--
    Top camera
  -->
  <node pkg="nao_driver" type="nao_camera.py" name="nao_camera1" required="true" args="--pip=$(arg nao_ip) --pport=$(arg nao_port)" output="screen">
	<rosparam command="load" file="$(find nao_behavior_tree)/launch/parameters.yaml"/>
    <remap from="image_raw" to="image_raw1"/>
  </node>
  
  <node pkg="nao_behavior_tree" type="color_filter_calibration" name="color_filter_calibration1" required="true" args="1" output="screen">
  	<rosparam command="load" file="$(find nao_behavior_tree)/launch/parameters.yaml"/>
  </node>

</launch>
